src/main/java/org/killbill/billing/client/model/InvoiceItem.java

```diff
@@ -52,6 +52,10 @@
     private String currency;
     private List<InvoiceItem> childItems;
 
+    private BigDecimal rate;
+    private Integer quantity;
+    private String itemDetails;
```

Comment:
@sbrossie Maybe we should eventually expose a proper object instead (i.e. introduce a `InvoiceItemDetailsJson` in jaxrs and even change `InvoiceItem` in api).

On the plus side, this allows us to leave it opaque for now until we finalize what it contains (very similar use-case to the metadata event field).